rclcpp/src/rclcpp/waitable.cpp

```diff
@@ -34,6 +34,12 @@ Waitable::get_number_of_ready_clients()
   return 0u;
 }
 
+size_t
+Waitable::get_number_of_ready_events()
+{
+  return 0u;
```

Comment:
This looks like it needs a `//TODO` marker

Reply:
In this implementation, waitables will not have any events, since it was stripped.

Reply:
Let's leave `QOSEventHandlerBase`/`QOSEventHandler` as a `Waitable`, since they are new classes. The `Waitable` interface will then continue to need a `get_number_of_ready_events()` member function.